addons/web/static/src/core/emoji_picker/emoji_picker.js

```diff
@@ -104,15 +106,18 @@ export function useEmojiPicker(ref, props, options = {}) {
 }
 
 export const loader = {
-    loadEmoji: () => loadBundle("web.assets_emoji"),
+    loadEmoji(env) {
+        return loadBundle("web.assets_emoji");
+    },
```

Comment:
We should probably consider creating the cors route for emoji directly in web to avoid all these env passing